webpack/loaders.js

```diff
@@ -23,14 +18,18 @@ export default (appName) => {
                 'transform-runtime',
                 'add-module-exports',
             ],
-            presets,
+            presets: [
```

Comment:
It has been problematic when we worked on a react-native project which required a specific babelrc to work. Having the conf in webpack allowed us to make all apps (react, ng-admin and react-native) work